ui/app/AppLayouts/Profile/panels/ProfileShowcasePanel.qml

```diff
@@ -70,20 +96,65 @@ DoubleFlickableWithFolding {
             width: ListView.view.width
             title: qsTr("In showcase")
             folded: root.flickable1Folded
-            rightAdditionalComponent: VisibilityDropAreaButtonsRow {
+            rightAdditionalComponent: Loader {
+                readonly property bool isDropAreaVisible: root.flickable1Folded && d.isAnyHiddenDragActive
+
                 width: d.additionalHeaderComponentWidth
                 height: d.additionalHeaderComponentHeight
-                margins: 0
-                visible: root.flickable1Folded &&
-                         (d.isAnyHiddenDragActive ||
-                          parent.containsDrag ||
-                          everyoneContainsDrag ||
-                          contactsContainsDrag ||
-                          verifiedContainsDrag)
+                sourceComponent: isDropAreaVisible && d.limitReached ? limitReachedHeaderButton :
+                                                                       isDropAreaVisible ? dropHeaderAreaComponent : counterComponent
+            }
```

Comment:
There is already a loader in the header. WDYT of just swapping the component instead of creating another one?
           
           rightAdditionalComponent: isDropAreaVisible && d.limitReached ? limitReachedHeaderButton :
                                                                        isDropAreaVisible ? dropHeaderAreaComponent : counterComponent

Reply:
Good point! Done!